### PR TITLE
Update the AD Integration Test command for build workflows

### PR DESCRIPTION
**Pull request by gaiksaya**
*Issue #, if available:* https://github.com/opendistro-for-elasticsearch/opendistro-build/issues/476
P41846943 

*Description of changes:*
Update the AD Integration Test command for release workflows as per the above issue

*Test Results:*
No way to test it right now since we don't have ODFE 1.12.0 built yet.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/staging-build-docker.yml [hunk 1]
```diff
@@ -348,8 +348,8 @@ jobs:
         run: |
           .github/scripts/setup_runners_service.sh docker --es
           export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../anomaly-detection; pwd
-          ./gradlew :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin 
-          
+          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin  
+
```
Comment (peterzhuamazon) [inline .github/workflows/staging-build-docker.yml:352]: Additional trailing spaces

File: .github/workflows/staging-build-rpm.yml [hunk 2]
```diff
@@ -333,8 +333,8 @@ jobs:
         run: |
           .github/scripts/setup_runners_service.sh rpm --es
           export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../anomaly-detection; pwd
-          ./gradlew :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin
-  
+          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin  
+
```
Comment (peterzhuamazon) [inline .github/workflows/staging-build-rpm.yml:337]: Spaces

File: .github/workflows/staging-build-tar.yml [hunk 3]
```diff
@@ -286,8 +286,8 @@ jobs:
         run: |
           .github/scripts/setup_runners_service.sh zip --es
           export PATH=$JAVA_HOME:$PATH; cd $GITHUB_WORKSPACE/../anomaly-detection; pwd
-          ./gradlew :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername=es-integrationtest -Dhttps=true -Duser=admin -Dpassword=admin
-  
+          ./gradlew integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin  
+
```
Comment (peterzhuamazon) [inline .github/workflows/staging-build-tar.yml:290]: spaces

File: .github/workflows/staging-build-windows.yml [hunk 4]
```diff
@@ -268,7 +268,7 @@ jobs:
           echo "running tests"
           cd ..\anomaly-detection
           dir
-          .\gradlew.bat :integTestRunner --tests "com.amazon.opendistroforelasticsearch.ad.rest.*IT" --tests "com.amazon.opendistroforelasticsearch.ad.e2e.*IT" -D tests.rest.cluster=localhost:9200 -D tests.cluster=localhost:9200 -D tests.clustername=es-integrationtest -D https=true -D user=admin -D password=admin --stacktrace
+          .\gradlew.bat integTest -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="es-integrationtest" -Dhttps=true -Duser=admin -Dpassword=admin
```
Comment (peterzhuamazon) [inline .github/workflows/staging-build-windows.yml:271]: spaces

**Review by peterzhuamazon — APPROVED** [review]

**Review by weicongs-amazon — APPROVED** [review]

**Pull request merged**